src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -192,3 +206,31 @@ export const ButtonGroupWithHelpAndError: Story = {
     </RadioGroup>
   )
 };
+
+export const ButtonGroupDisabled: Story = {
+  name: 'Button group disabled',
+  render: (args) => (
+    <RadioGroup buttonGroup {...args}>
```

Comment:
same re: testing w/ disabled on fieldset vs inputs

Reply:
Tested! Works for individual radios, radio buttons and fieldset.